Client/src/components/Pages/Home/home.jsx

```
@@ -1,23 +1,471 @@
+    const scores = [85, 76, 92, 64, 88, 71];
+    
+    return Array(count).fill().map((_, index) => ({
+      id: index,
+      title: `Ad Campaign ${index + 1}`,
+      imageUrl: `/api/placeholder/400/225`, // Using a better aspect ratio for thumbnails
+      // Example video URL
+      duration: "0:30",
+      brand: `Brand ${index % 5 + 1}`,
+      engagementScore: scores[index % scores.length],
+      dominantEmotion: emotions[index % emotions.length],
+      viewCount: parseInt(cryptoRandomString({length: 4, type: 'numeric'}), 10) + 1000 
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)